tests/testthat/test-scale.R

```diff
@@ -88,9 +88,11 @@ test_that("na_rm argument works for step_scale", {
   mtcars_na <- mtcars
   mtcars_na[1, 1:4] <- NA
 
+  expect_warning({
```

Comment:
Please switch to use `expect_snapshot()` so that we can verify that it is the correct warning that is omitted 😄

Reply:
Done!